src/SMG/ManagerBundle/Controller/ManagersController.php

```diff
@@ -166,4 +165,55 @@ public function patchUserEnableAction(User $user)
 
         $this->get('fos_user.user_manager')->updateUser($user);
     }
+
+    /**
+     * @param User    $user
+     * @param Request $request
+     *
+     * @Annotations\Put("/users/{id}/password")
+     *
+     * @return Response
+     */
+    public function putUserPasswordAction(User $user, Request $request)
+    {
+        $this->throwIfClientNot('backend');
+
+        $requestData = $this->requestIsJsonWithKeysOrThrow(
+            $request,
+            ['new_password']
+        );
+
+        $user->setPlainPassword($requestData['new_password']);
```

Comment:
that would mean anyone with a backend token (so anyone) can change the password of anyone else as long as they know this URL / the user ID. That's why I mentioned I could give the current user id + token to verify he has ROLE_ADMINPANEL